use-timescale/migration/troubleshooting.md

```diff
@@ -0,0 +1,196 @@
+- compression policies
+- continuous aggregate refresh policies
+
+If you would like to use concurrency nonetheless, we recommend that you disable
+all background jobs in the source database before running `pg_dump`, and
+re-enable them once the dump is complete. If the dump procedure takes longer
+than the continuous aggregate refresh policy's window, you will need to
+manually refresh the continuous aggregate in the correct time range. For more
+information, consult the [refresh policies documentation][refresh-policies].
+
+To disable the jobs:
+```sql
+SELECT public.alter_job(id::integer, scheduled=>false)
+FROM _timescaledb_config.bgw_job
+WHERE id >= 1000; 
+```
+
+To re-enable the jobs:
+```sql
+SELECT public.alter_job(id::integer, scheduled=>true)
+FROM _timescaledb_config.bgw_job
+WHERE id >= 1000; 
+```
+
+[refresh-policies]: https://docs.timescale.com/use-timescale/latest/continuous-aggregates/refresh-policies/
+
+## Restoring with concurrency
+
+If the directory format is used for pg_dump and pg_restore, concurrency can be
+employed to speed up the process. Unfortunately, loading the tables in the 
+`timescaledb_catalog` schema concurrently will cause errors. Furthermore, the 
+`tsdbadmin` does not have sufficient privileges to disable triggers in this 
+schema. To get around this limitation, load this schema serially, and then load 
+the rest of the database concurrently.
+
+```bash
+# first, load JUST the _timescaledb_catalog serially
+pg_restore -d "$TARGET" \
+    --format=directory \
+    --schema='_timescaledb_catalog' \
+    --exit-on-error \
+    --no-tablespaces \
+    --no-owner \
+    --no-privileges \
+    dump
+
+# next, load everything EXCEPT the _timescaledb_catalog with concurrency
+pg_restore -d "$TARGET" \
+    --format=directory \
+    --jobs=8 \
+    --exclude-schema='_timescaledb_catalog' \
+    --exit-on-error \
+    --disable-triggers \
+    --no-tablespaces \
+    --no-owner \
+    --no-privileges \
+    dump
+```
+
+## Ownership of background jobs
+
+The `_timescaledb_config.bgw_jobs` table is used to manage background jobs. This
+includes both user-defined actions, compression policies, retention policies,
+and continuous aggregate refresh policies. On Timescale, this table has a
+trigger which ensures that no database use can create or modify jobs owned by
+another database user. This trigger can provide an obstacle for migrations.
+
+If the `--no-owner` flag is used with pg_dump and pg_restore, all objects in the
+target database will be owned by the user that ran pg_restore, likely tsdbadmin.
+If all the background jobs in the source database were owned by a user of the
+same name as the user running pg_restore (again likely tsdbadmin), then loading
+the `_timescaledb_config.bgw_jobs` table will work. If the background jobs in 
+the source database were not owned by this user, and especially if they were
+owned by the postgres user, restoring into this table will fail due to the
+trigger.
```

Comment:
Jobs would still throw permission error: https://github.com/timescale/timescaledb/issues/5745

Reply:
Just to clarify: that issue only arises when using custom roles in the source db?

Reply:
Yeah. That issue is, you have a job owned by user X the table is owned by tsdbadmin. The job fails because it's being executed as X and it requires that X owns the underlying tables. Trying to change ownership from tsdbadmin to X on the table fails.

Last time I tried it was still failing.

Reply:
Is this more correct/clear?